pom.xml

```
@@ -183,8 +183,8 @@
                         <compilerArgument>-Xlint:deprecation</compilerArgument>
                         -->
                         <fork>true</fork>
-                        <source>1.8</source>
-                        <target>1.8</target>
+                        <source>17</source>
+                        <target>17</target>
```

```suggestion
                        <release>17</release>
```

https://maven.apache.org/plugins/maven-compiler-plugin/examples/set-compiler-release.html